diff --git a/node_modules/babel-plugin-jest-hoist/build/index.js b/node_modules/babel-plugin-jest-hoist/build/index.js
index f02c568..b1694d6 100644
--- a/node_modules/babel-plugin-jest-hoist/build/index.js
+++ b/node_modules/babel-plugin-jest-hoist/build/index.js
@@ -131,6 +131,7 @@ FUNCTIONS.mock = args => {
           invariant(
             (scope.hasGlobal(name) && WHITELISTED_IDENTIFIERS.has(name)) ||
             /^mock/i.test(name) || // Allow istanbul's coverage variable to pass.
+            /^_get__/i.test(name) ||
               /^(?:__)?cov/.test(name),
             'The module factory of `jest.mock()` is not allowed to ' +
               'reference any out-of-scope variables.\n' +
