diff --git a/node_modules/babel-plugin-jest-hoist/build/index.js b/node_modules/babel-plugin-jest-hoist/build/index.js
index f02c568..aa9e488 100644
--- a/node_modules/babel-plugin-jest-hoist/build/index.js
+++ b/node_modules/babel-plugin-jest-hoist/build/index.js
@@ -131,18 +131,19 @@ FUNCTIONS.mock = args => {
           invariant(
             (scope.hasGlobal(name) && WHITELISTED_IDENTIFIERS.has(name)) ||
             /^mock/i.test(name) || // Allow istanbul's coverage variable to pass.
+              /^_get__/i.test(name) ||
               /^(?:__)?cov/.test(name),
           );
         }
       }